connectors/snmp-connector/clients/snmpClient.go

```diff
@@ -109,14 +119,21 @@ func setup(target string, secData *utils.SecurityData) (*snmp.GoSNMP, error) {
 
 func setupV2c(target string, community *utils.SecurityData) (*snmp.GoSNMP, error) {
 	err := validate(target, community)
+
 	if err != nil {
 		log.Error("|snmpClient.go| : [setupV2c] : Failed to setup snmp v3: ", err)
 		return nil, errors.New("validation failed")
 	}
-	goSnmp := snmp.Default
-	goSnmp.Target = target
-	goSnmp.Community = community.Name
-	return goSnmp, nil
+
+	return &snmp.GoSNMP{
+		Target:         target,
+		Port:           161,
+		Community:      community.Name,
+		Version:        snmp.Version2c,
+		Timeout:        time.Duration(2) * time.Second,
```

Comment:
Can we add Duration and Timeout to V3 structs too?

Reply:
Yep